Packs/PAN-OS/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "PAN-OS",
     "description": "Manage Palo Alto Networks Firewall and Panorama. For more information see Panorama documentation.",
     "support": "xsoar",
-    "currentVersion": "1.6.19",
+    "currentVersion": "1.6.20",
```

Comment:
lets bump it in a ```minor``` bump instead of ```revision``` as this is quite a big change.

Reply:
Would that be 1.7.1 or 1.7.0?

Reply:
it should be 1.7.0 after solving the conflicts

Reply:
Well, sorry I misled you, apparently this PR wasn't updated against the master for a long time.
We already have 1.8.8 at our master version, so 1.9.0 is good.